build.gradle

```
@@ -107,23 +139,55 @@ List<String> resolveIntelliJVersions(String versionKey) {
   return [versionValue]
 }
 
-class ResolveIntelliJVersions extends DefaultTask {
-  @Input
-  @Option(option = 'version-key', description = 'Either release number (like 2020.3) or key of intellijVersions (like eapOfLatestSupportedMajor)')
-  String versionKey
-
-  @OptionValues('version-key')
-  List<String> getAvailableVersionKeys() {
-    return new ArrayList<>(project.intellijVersions.keySet())
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import org.jsoup.nodes.Document
+
+String checkForEapForFutureRelease(String intellijMajorVersion) {
```

For moar clarity:
```suggestion
String checkForEapForFutureRelease(String intellijMajorPartOfBuildNumber) {
```
(at least I suppose that's what's expected here!)